.scalafmt.conf

```diff
@@ -1,2 +1,15 @@
-version = 2.2.2
+version = "2.2.2"
 maxColumn = 140
+align = most
+continuationIndent.defnSite = 2
+assumeStandardLibraryStripMargin = true
+docstrings = JavaDoc
+lineEndings = preserve
+includeCurlyBraceInSelectChains = false
+danglingParentheses = true
+spaces {
+  inImportCurlyBraces = true
+}
+optIn.annotationNewlines = true
+
+rewrite.rules = [SortImports, RedundantBraces]
```

Comment:
I'd prefer to keep the current formatting rules, that is - default except for `maxColumn`. We use them everywhere

Reply:
Yep, no problem. It's just that when I'm working I prefer these rules but I'll remove them if my design arrives to something interesting. 🙂

Reply:
No worries, thanks :)